src/averageAges.js

```diff
@@ -13,13 +13,26 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+
+function peopleAgeArray(object) {
```

Comment:
peopleAgeArray expected on property object?)
and after you for object use array methods )